lib/Log.js

```diff
@@ -1,104 +1,77 @@
 var fs    = require('fs'),
     chalk = require('chalk'),
-    spawn = require('child_process').spawn;
+    CLI   = require('./CLI.js');
 
-var Log = module.exports = {
-  count: 0,
-  prev: ''
-};
-
-// Empty line.
-var re_blank = /^[\s\r\t]*$/;
-
-/**
- * Styles of title and content.
- * @type {{def: {title: string, content: string}, out: {title: string, content: string}, err: {title: string, content: string}}}
- */
-Log.styles = {
-  def   : {title: 'blue', content: 'grey'},
-  entire: {title: 'green', content: 'black'},
-  out   : {title: 'green', content: 'black'},
-  err   : {title: 'red', content: 'red'}
-};
-
-Log.colors = ['black', 'yellow', 'magenta', 'cyan', 'grey'];
+var Log = module.exports = {};
 
 /**
  * Tail logs from file stream.
- * @param {String|Object} path
- * @param {String} title
+ * @param {Object} apps_list
  * @param {Number} lines
- * @returns {*}
+ * @param {Boolean} raw
+ * @param {Function} callback
+ * @return
  */
-Log.stream = function(path, title, lines, raw){
-  var type = 'def';
 
-  // Make options in the right position.
-  if (typeof path == 'object') {
-    type = path.type || type;
-    path = path.path;
-  }
-  if (typeof title == 'number') {
-    raw = lines;
-    lines = title;
-    title = null;
-  }
-
-  if(this.prev != title){
-    this.prev = title;
-    this.count ++;
-    if(this.count >= this.colors.length){
-      this.count = 0;
+Log.tail = function(apps_list, lines, raw, callback) {
+  if (lines === 0 || apps_list.length === 0)
+    return callback && callback();
+
+  var count = 0;
+
+  apps_list.forEach(function(app, index) {
```

Comment:
The apps_list still should be sorted by `mtime` of `app.path`, otherwise the #1109 issue still exists, what do you think?

SOL:
```javascript
!!apps_list.length && apps_list.sort(function(a, b){
  return (fs.existsSync(a.path) ? fs.statSync(a.path).mtime.valueOf() : 0) -
(fs.existsSync(b.path) ? fs.statSync(b.path).mtime.valueOf() : 0);
}).forEach('[YOUR CODE GOES HERE]');
```